docs/IdentityCard.md

```diff
@@ -0,0 +1,12 @@
+# IdentityCard
+
+- `v::identityCard()`
```

Comment:
I think this name is too generic to cover only Polish Identify Card, do you have a better name in mind?